packages/cli/src/commands/org/resources/pull.ts

```diff
@@ -64,6 +62,28 @@ export default class Pull extends Command {
       description: 'Overwrite resources directory if it exists.',
       default: false,
     }),
+    resourceTypes: flags.build<ResourceSnapshotType>({
```

Comment:
Using [custom flags](https://oclif.io/docs/flags#custom-flags) to preprocess the input and convert it to a type that can easily be manipulated later in the code. It also removes the need for casting a flag to a Custom type.